js/search/searchUtils.js

```diff
@@ -10,6 +10,9 @@ const { isMac } = require('../utils/misc.js');
 /*eslint class-methods-use-this: ["error", { "exceptMethods": ["checkFreeSpace"] }] */
 class SearchUtils {
 
+    constructor() {
+        this.indexVersion = searchConfig.INDEX_VERSION;
```

Comment:
Do we really need the constructor if we can already get the version value from `searchConfig.INDEX_VERSION`?